clients/catalyst.go

```diff
@@ -45,10 +45,9 @@ type OutputLocation struct {
 }
 
 type OutputsRequest struct {
-	SourceMp4          bool `json:"source_mp4"`
-	SourceSegments     bool `json:"source_segments"`
-	TranscodedSegments bool `json:"transcoded_segments"`
-	AutoMP4            bool `json:"auto_mp4"`
+	SourceMp4 bool   `json:"source_mp4"`
+	HLS       string `json:"hls"`
+	MP4       string `json:"mp4"`
```

Comment:
Wow is this already supported?

Reply:
Yeah, in the other PR: https://github.com/livepeer/catalyst-api/pull/546